project_event/models/project_task.py

```diff
@@ -207,14 +210,30 @@ class Task(models.Model):
     )
     spectators = fields.Char(
         string='Spectators',
-     )
+        size=10,
+        default='-',
+    )
 
     @api.onchange('spectators')
-    def check_numeric(self):
-        try:
-            int(self.spectators)
-        except Exception as exception:
-            raise ValidationError(_('The Input value should be numeric'))
+    def onchange_spectators(self):
+        if self.spectators:
+            if self.spectators == '-':
+                return
+            try:
+                int(self.spectators)
+            except Exception as exception:
```

Comment:
we dont need as exception if you will not use it

Reply:
@rimbendhaou  Ok